src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Bmi2.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel BMI2 hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Bmi2
+    public abstract class Bmi2
```

Comment:
(for my knowledge)  `Bmi2` shouldn't inherit from `Bmi1`?

Reply:
nvm, I see it the discussion in the PNS file

---
In reply to: [206531480](https://github.com/dotnet/coreclr/pull/19186#discussion_r206531480) [](ancestors = 206531480)